_includes/content.html

```diff
@@ -94,28 +105,22 @@
 {% highlight ruby %}
 Griddler.configure do |config|
   config.processor_class = EmailProcessor # MyEmailProcessor
-  config.to = :token # :full, :email, :hash
-  # :raw    => 'AppName <s13.6b2d13dc6a1d33db7644@mail.myapp.com>'
-  # :email  => 's13.6b2d13dc6a1d33db7644@mail.myapp.com'
-  # :token  => 's13.6b2d13dc6a1d33db7644'
-  # :hash   => { raw: '', email: '', token: '', host: '' }
+  config.processor_method = :process
```

Comment:
I think I explained it here, let me know if it's still unclear: https://github.com/thoughtbot/griddler/commit/6a67ff1583f14fddb054e58525ed49885fc24636